README.md

```diff
@@ -4,13 +4,11 @@
 
 👋 **Welcome!**
 
-We’ve assembled a toolkit that university instructors can use to easily prepare labs, homework, or classes. The content is self-contained so that it can be easily incorporated into an existing curriculum. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
+We’ve assembled a toolkit that Hugging Face ambassadors can use to easily prepare workshops, events, labs or classes. The content is self-contained so that it can be easily taken over. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
```

Comment:
I think we wanted both ambassadors and instructors to use this. Maybe let's just say "that anyone can use to..." and then this can be aimed for any audience

Reply:
Ok so will could have => 

We’ve assembled an education toolkit that anyone can use to easily prepare workshops, events, labs or classes. The content is self-contained so that it can be easily taken over. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).